src/SFA.DAS.ApprenticeFeedback.Web/Pages/ExitInterview/Question4.cshtml.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeFeedback.Infrastructure.Session;
+using SFA.DAS.ApprenticeFeedback.Web.Filters;
+using SFA.DAS.ApprenticeFeedback.Web.Services;
+using SFA.DAS.ApprenticePortal.Authentication;
+using SFA.DAS.ApprenticePortal.SharedUi.Menu;
+using System.ComponentModel.DataAnnotations;
+
+namespace SFA.DAS.ApprenticeFeedback.Web.Pages.ExitInterview
+{
+    [HideNavigationBar]
+    public class Question4Model : ExitInterviewContextPageModel, IHasBackLink
+    {
+        [BindProperty]
+        [Required(ErrorMessage = "Please select an answer")]
+        public string RemainedReason { get; set; }
+        public string[] Reasons = new[]
+        {
+            "I had more support and guidance from my employer",
+            "I had more support and guidance from my training provider",
+            "I had more information on how it would be assessed at end point assessment",
+            "My employer did not end my apprenticeship",
+            "My training provider did not end my apprenticeship",
+            "The employer gave me more time to undertake learning and training",
+            "The employer provided me with the training they were meant to",
+            "The training provider delivered better quality training",
+            "There was a mentor or learning coach within or outside of my organisation",
+        };
+
+        public string Backlink => (ExitInterviewContext.CheckingAnswers) ? $"./checkyouranswers" : $"./question3";
+
+        public Question4Model(IExitInterviewSessionService sessionService)
+            : base(sessionService)
+        {
+        }
+
+        public IActionResult OnGet([FromServices] AuthenticatedUser user)
+        {
```

Comment:
Potential bug here, whereby if on the check your answers page they select back, then on question 4 they select back. the session will still have the CheckAnswers value set to true so on forward would skip question 4.

Unsure how best to go about that, other than not having a back link on the check your answers page?

Reply:
The user could use the back button on the browser to achieve the same, so I'm not sure removing the back link is the answer?